LaunchUpCorp/Dashboard-Builder
# feat: dashboards list sidebar with search & add new dashboard

## created a sidebar with filter search & add new dashboard functionality

### Type of Change


### Description

1) a sidebar display a list of dashboards
2) a search & filter dashboard functionality,  based on keywords of names
3) a add new dashboards to the list functionality
4) storybook which shows 3 components and overall interactions

### Testing Instructions

1) 'npm run dev' to view the sidebar
2) 'npm run storybook' to see the storybook


### Screenshots

<img width="894" alt="image" src="https://github.com/LaunchUpCorp/Dashboard-Builder/assets/105130114/337c6893-9270-4316-8909-96d34eef454f">
<img width="804" alt="image" src="https://github.com/LaunchUpCorp/Dashboard-Builder/assets/105130114/ebaa9e5f-61aa-4243-aab8-5feb289d105e">
<img width="897" alt="image" src="https://github.com/LaunchUpCorp/Dashboard-Builder/assets/105130114/2544ef8f-4203-4cc8-ba02-d2d21120cb08">
